Applications/Utils/MeshEdit/Vtu2Grid.cpp

```diff
@@ -0,0 +1,225 @@
+/**
+ *
+ * @copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/LICENSE.txt
+ */
+
+#include <tclap/CmdLine.h>
+
+#include "Applications/ApplicationsLib/LogogSetup.h"
+#include "InfoLib/GitInfo.h"
+
+#include "MeshLib/IO/writeMeshToFile.h"
+#include "MeshLib/Mesh.h"
+#include "MeshLib/MeshEditing/RemoveMeshComponents.h"
+#include "MeshLib/MeshGenerators/MeshGenerator.h"
+#include "MeshLib/MeshSearch/ElementSearch.h"
+
+#include <vtkAbstractArray.h>
+#include <vtkCellData.h>
+#include <vtkCellLocator.h>
+#include <vtkDataArray.h>
+#include <vtkDoubleArray.h>
+#include <vtkIntArray.h>
+#include <vtkSmartPointer.h>
+#include <vtkUnstructuredGrid.h>
+#include <vtkXMLUnstructuredGridReader.h>
+
+static std::string cell_id_name = "CellIds";
```

Comment:
Why not:
```suggestion
std::string const cell_id_name = "CellIds";
```
?

Reply:
Can you try
```suggestion
constexpr std::string_view cell_id_name = "CellIds";
```
... please?

Reply:
That doesn't work.

> Konvertierung von Argument 1 von "const std::string_view" in "const std::string &" nicht möglich

Reply:
sure, the function signatures where the `string_view` is passed to have to be changed too.
If this works, then we can start using the `string_view` in the whole project.